app/models/user.rb

```diff
@@ -8,10 +8,19 @@ class User < ApplicationRecord
   validates :password, confirmation: true
 
   enum role: [:user, :admin]
-  
+
   def all_books
      books.includes(:user_books)
       .select(:id, :guten_id, :author, :title, :current_page)
       .order(id: :asc)
   end
+
+  def update_settings(params)
+        self.update(
+                 music_genre: params[:music_genre],
+                 dyslexic_font: params[:dyslexic_font],
+                 dark_mode: params[:dark_mode],
+                 font_size: params[:font_size]
+                )
```

Comment:
To avoid hard coding here, lines 19-24 could be replaced with:
```
update(params.permit(:music_genre, :dyslexic_font, :font_size, :dark_mode))
```
or
```
settings_params = params.permit(:music_genre, :dyslexic_font, :font_size, :dark_mode)
update(settings_params)
```
I like the first one a little better, but perhaps the 2nd one is a little more readable.

Reply:
This reminds me... i was just going to pass in the user_params straight to the update method. 

I updated this in the GET users endpoints.